packages/strapi-plugin-i18n/admin/src/permissions.js

```diff
@@ -0,0 +1,8 @@
+const i18nPermissions = {
+  create: [{ action: 'plugins::i18n.locale.create', subject: null }],
+  delete: [{ action: 'plugins::i18n.locale.delete', subject: null }],
+  update: [{ action: 'plugins::i18n.locale.update', subject: null }],
+  read: [{ action: 'plugins::i18n.locale.refead', subject: null }],
+};
```

Comment:
Can you add another key to permissions `accessMain: [{ action: 'plugins::i18n.locale.read', subject: null}, { action: 'plugins::i18n.locale.create, subject: null}]` and the add the following to the index.js:

```
settings: {
  global: {
    links: [
      {
        title: {
          id: getTrad('plugin.name'),
          defaultMessage: 'Internationalization',
        },
        name: 'internationalization',
        to: `${strapi.settingsBaseURL}/internationalization`,
        Component: () => <SettingsPage />,
        permissions: pluginPermissions.accessMain,
      },
    ],
  },
```

By doing so, if a user does not have the permission to either create or read the settings the link won't be displayed in the settings menu.